main.go

```diff
@@ -225,34 +200,40 @@ func main() {
 }
 
 func prepareFilter(filters []string) (tracee.Filter, error) {
-
 	filterHelp := "\n--filter allows you to specify values to match on for fields of traced events.\n"
-	filterHelp += "The following options are currently supported:\n"
-	filterHelp += "uid: only trace processes or containers with specified uid(s).\n"
+	filterHelp += "\n"
+	filterHelp += "uint filters use the following operators: '=', '!=', '<', '>'\n"
```

Comment:
I think it's better to rename "uint filter" to "numerical filter" it doesn't matter the actual type, just that it's a number and can be treated as a number in the filter predicate